File: app/src/main/java/com/hhh/onepiece/live/LiveViewHolder.java
package com.hhh.onepiece.live;

import android.graphics.drawable.ColorDrawable;
import android.view.View;

import com.hhh.mvvm.recycler.RecyclerViewHolder;
import com.hhh.onepiece.R;
import com.hhh.onepiece.model.WorksModel;
import com.hhh.onepiece.view.NetworkImageView;

import androidx.annotation.NonNull;

public class LiveViewHolder extends RecyclerViewHolder<WorksModel> {

  private NetworkImageView mCoverView;

  public LiveViewHolder(@NonNull View itemView) {
    super(itemView);
    mCoverView = findViewById(R.id.cover);
  }

  @Override
  public void bind(@NonNull WorksModel worksModel) {
    super.bind(worksModel);
    mCoverView.getHierarchy().setPlaceholderImage(new ColorDrawable(worksModel.mColor));
  }
}

File: app/src/main/java/com/nb/core/test/live/LiveViewHolder.kt
package com.nb.core.test.live

import android.graphics.drawable.ColorDrawable
import android.view.View
import com.facebook.drawee.view.SimpleDraweeView
import com.hhh.onepiece.R
import com.nb.core.RecyclerViewHolder
import com.nb.core.test.model.WorksModel

class LiveViewHolder(view: View) : RecyclerViewHolder<WorksModel>(view) {

    private val coverView: SimpleDraweeView = itemView.findViewById(R.id.cover)

    override fun bind(model: WorksModel) {
        super.bind(model)
        coverView.hierarchy.setPlaceholderImage(ColorDrawable(model.color))
    }
}